### DISCUSSION TITLE
excessive use of "Load diff" deferred diff loading

### DISCUSSION THREAD

**Discussion by vasslitvinov** [community/community#56957] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

I made a PR that touches 300+ files. In the "File changed" tab I can see the diffs only on the first half of them, the rest shows "Load diff". So if I want to review all my changes, I need to click "Load diff" >100 times. This is highly unreasonable and counter-productive. At the very least there should be a conspicuous button to "load all diffs" and/or "load all small diffs". Thanks!

**Comment by fiendish** [comment]
Please, yes.
